### DISCUSSION TITLE
part-ii/2021-136/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#331] [Question comments]
Part II 2021 Riemann Surfaces: Paper 2, Section II, F 

https://questions.tripos.org/part-ii/2021-136/

**Comment by artie2000** [comment]
The word *homotopic* is sneaky but important.

<details>
<summary>The example:</summary>

$\sqrt{\sqrt{z}+1}$ around $z=1$.

Start with a function element (on $D(1,1)$, say) given by the +ve choice of both roots. This is easy to analytically continue to any point: extend the branches of $\sqrt{}$ to $C_\star$ minus a half-line. Looping once around 0, the inner choice of root becomes negative; looping around 1 then gives a different outer choice of root, yet a loop around 1 is null-homotopic in $C_\star$.

</details>
